demos/super_resolution_demo/main.cpp

```diff
@@ -53,6 +48,37 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
     return true;
 }
 
+
+template <typename T>
+void matToBlob(const cv::Mat& orig_image, InferenceEngine::Blob::Ptr& blob, int batchIndex = 0) {
```

Comment:
I think `matU8ToBlob` from common should be updated instead of introducing similar function.

Reply:
@Wovchena Updated `matU8ToBlob`, please take a look at it.